.github/workflows/r-ci.yml

```diff
@@ -91,6 +87,13 @@ jobs:
       - name: Update Packages
         run: Rscript -e 'update.packages(ask=FALSE)'
```

Comment:
I recommend to remove this block.  It would allow you to remove the next block (ie lines 90 to 95 as well).  And cut CI time down by a few minutes.

/CC @mojaveazure

Reply:
@eddelbuettel trying commit #3 https://github.com/single-cell-data/TileDB-SOMA/pull/1996/commits/de92e139b12dad908df1bf4da827a328143d0f7e

Reply:
@eddelbuettel:

* Commit 2 https://github.com/single-cell-data/TileDB-SOMA/pull/1996/commits/be8c808c6468d8b2ce446cf8735b5ff35f122d00 had green CI
* There were 3 of 3 CI fails with commit number 3: https://github.com/single-cell-data/TileDB-SOMA/pull/1996/commits/de92e139b12dad908df1bf4da827a328143d0f7e
* Commit 4 https://github.com/single-cell-data/TileDB-SOMA/pull/1996/commits/5a76da15cd3a88e51370720fdba0df77dd3ace74 reverts to commit 2 and has green CI

Reply:
If I look at the correct page not three for three.  One, plus two cancels. We know coverage is more fragile.

https://github.com/single-cell-data/TileDB-SOMA/actions/runs/7449708671/job/20267723363

Reply:
@eddelbuettel I watched each one.

* Commit 3 had three fails
* Commit 4 had a cancel (I don't know why), then a successful re-run

Reply:
@eddelbuettel I am happy to re-run the already-green CI another time for commit number 4.

Reply:
Am I looking in the wrong place?

![image](https://github.com/single-cell-data/TileDB-SOMA/assets/673121/c15f912e-8cc0-44b2-b7ae-18c62ff0edae)

And 

![image](https://github.com/single-cell-data/TileDB-SOMA/assets/673121/d130166a-4d72-4f41-b03f-94d694ffc738)

Anyway, I have no beef with this but it may be a good idea to get someone else to look at this.